charts/backstage/values.yaml

```diff
@@ -35,6 +35,9 @@ commonLabels: {}
 # -- Annotations to add to all deployed objects
 commonAnnotations: {}
 
+# -- Annotations to add to the backend deployment pods
+podAnnotations: {}
```

Comment:
Since this value is applied only to the Pod deployed via Backstage Deployment, can you nest this value under `.backstage` root key please (line [81](https://github.com/backstage/charts/pull/54/files#diff-8060d9a38501197eddb026fced7fe56f104f4f5143210c6d47781b831a7097f4L81) and below)? That way we can keep things organized a bit.

Reply:
done, looking much better now